apps/explorer/src/modules/tokens/components/TokenTable.vue

```diff
@@ -258,7 +258,8 @@ export default class TokenTable extends Vue {
   }
 
   get loading() {
-    return this.$apollo.loading || this.syncing
+    // return this.$apollo.loading || this.syncing
+    return true
```

Comment:
@olgakup be careful as you have overridden state (restore previous behavior)

Reply:
fixed in [5df127..](https://github.com/EthVM/EthVM/pull/729/commits/5df127701deb93c334add58d3c870dd6b9019f90)